pkg/apis/serving/v1/configuration_validation.go

```diff
@@ -76,8 +76,6 @@ func (c *Configuration) validateLabels() (errs *apis.FieldError) {
 		switch key {
 		case serving.RouteLabelKey:
 			// Known valid labels.
-		case serving.VisibilityLabelKey:
-			errs = errs.Also(validateClusterVisibilityLabel(val))
```

Comment:
Might this happen if a ksvc is labeled?

Reply:
Yes this happens, but the visibility label in configuration has no effect. So I wonder that the validation is not necessary in Configuration.

Reply:
So my concern is that the default case will blow up if we don't have this case

Reply:
Oh, that is correct. Thank you. I added `serving.VisibilityLabelKeyObsolete` to the valid label key. So the default case will not blow up it.